assets/js/modules/analytics-4/utils/tag-matchers.js

```diff
@@ -59,3 +59,5 @@ export function getTagMatchers() {
 
 	return tagMatchers;
 }
+
+export default getTagMatchers();
```

Comment:
This was added so it can be consumed by `assets/js/util/test/extractExistingTag.js` as simply as it did for the legacy `analytics` tags.